pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -302,16 +302,11 @@ func (pl *dynamicResources) PreEnqueue(ctx context.Context, pod *v1.Pod) (status
 	return nil
 }
 
-// isSchedulableAfterClaimChange is invoked for all claim events reported by
+// isSchedulableAfterClaimChange is invoked for add and update claim events reported by
```

Comment:
nit: to add

Reply:
Here it should be 'for', we call isSchedulableAfterClaimChange when we receive the add and update event.